source/_guides/6-php-client-intro.md

```diff
@@ -0,0 +1,207 @@
+# PHP client library intro
+
+
+### CMS Clients
+
+Note: If you just want to use rokka in a CMS like Drupal, Wordpress or Kirby, we recommend using the respective plugins (add links). If you want to know, how the PHP library works and what it provides, this intro may be a good start nevertheless.
+
+## Preparation
+
+If you already know the basics of rokka and have uploaded an image and created a stack, you can directly skip to the "The PHP library" chapter below.
+
+### Get a rokka account
+
+Before we start with the actual PHP stuff, we do some preparation. First, you of course need a rokka account. Head over to [the signup page](https://rokka.io/dashboard/#/signup) and get one. It's free for 90 days and afterwards still free for up to 5 GB of traffic and 2 GB of storage.
+
+After you signed up, you'll get an email with your api key, don't loose that.
+
+### Install the rokka CLI
+
+For some basic setup, we'll use the rokka CLI, go and [get it](https://github.com/rokka-io/rokka-go) and follow the instructions to install it.
```

Comment:
I'm still looking for a way to generate a direct download link but GitHub doesn't allow somehow to link to the latest release download item :/